mava/configs/logger/ff_ippo.yaml

```diff
@@ -0,0 +1,10 @@
+defaults:
+    - base_logger
+
+system_name: ff_ippo
+
+# --- Neptune options ---
+kwargs:
+  neptune_project: Instadeep/Mava
+  neptune_tag: [ff-ippo, rware]
+  name: ff-ippo
```

Comment:
For all of these we can keep name to None, we do not really use name on Nepune. We can rely on `system_name` for grouping etc.